docs/source/using/basic-usage.adoc

```diff
@@ -235,6 +252,16 @@ For example:
 $ minishift config unset memory
 ----
 
+[NOTE]
+====
+Precedence order for user defined values.
+
+. A user-defined flag.
```

Comment:
I would use plural forms here. Also, we need to consistently use (or not use) periods at the end of each list item. I also think "user-defined" is confusing here, and it would be more clear to simply state "command line flags."

```adoc
. Command line flags.
. Environment variables.
. Instance-specific configuration.
. Global configuration.
```